### ISSUE TITLE
Check wing length measures: P3 vs total wing length

### ISSUE THREAD

**Issue by StefanVriend** [SPI-Birds/pipelines#191]
Data owners may differ in what they measure as "wing length". In some cases, this may be the total wing length (from the carpal to the tip of the longest primary), whilst in other cases, this may refer to the length of the third primary (P3).

The new Measurement_data table in v1.2 makes it easy to record both, or any other new (wing) measure not previously come across in pipelines/datasets hosted at SPI-Birds.

**Comment by LiamDBailey** [comment]
If we do this for wing length should we also do the same for tarsus (instead of converting between Oxford min, Svensson & Svensson alt)?

**Comment by StefanVriend** [comment]
Yes, indeed. When converting the old pipelines to v1.2, I do that (see e.g. https://github.com/SPI-Birds/pipelines/blob/v1-2_pipelines/R/format_CHO.R). In some ases, I will double-check with the data owners what tarsus measure they use.
